### ISSUE TITLE
.coveragerc isn't combining the location of where tox on Windows is put

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8725]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)| @hawkowl reported|
|-|-|
|Trac ID|trac#8725|
|Type|defect|
|Created|2016-08-04 21:56:55Z|
|Branch|https://github.com/twisted/twisted/tree/8725-coverage-windows|

We need to add ```build/*/Lib/site-packages/twisted```

<details><summary>Searchable metadata</summary>

```
trac-id__8725 8725
type__defect defect
reporter__hawkowl hawkowl
priority__normal normal
milestone__None None
branch__8725_coverage_windows 8725-coverage-windows
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1470347815073985 1470347815073985
changetime__1470480185936021 1470480185936021
version__None None
owner__hawkowl hawkowl

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_1"></a>|@hawkowl commented|
|-|-|

PR: https://github.com/twisted/twisted/pull/462

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_3"></a>|@hawkowl commented|
|-|-|

Please review.

https://codecov.io/gh/twisted/twisted/src/062e39bd1d1ce3cc1ab921a52f804032aa18637a/twisted/internet/iocpreactor/reactor.py should be proof it's working, as there's an appveyor icon there and https://codecov.io/gh/twisted/twisted/tree/062e39bd1d1ce3cc1ab921a52f804032aa18637a contains just twisted and no build/ dir.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_4"></a>|@adiroiban set owner to @hawkowl|
|-|-|

Looks good. Thanks!

Please note that appveyor is not configured to build on push, only on PR, so it  will not report pushed to master.

We might want to run appveyour for all PRs and only to run on 'push' for master.

But this can be merged. Thanks!

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_5"></a>|GitHub set status to `closed`|
|-|-|

In changeset 88bae4e39cab4fca59a4fc01e2ad480653de9c23
```
#!CommitTicketReference repository="" revision="88bae4e39cab4fca59a4fc01e2ad480653de9c23"
Merge 8725-coverage-windows: Fix coverage reporting on Windows when using tox

Author: hawkowl
Reviewer: adiroiban
Fixes: #8725
```
